### PR TITLE
Add team to website

### PR DESCRIPTION
**Pull request by Schmavery**
I'll follow up with adding the expand/contract and the "Built By O(1) Labs" header.
 
image
image

### CODE REVIEW

File: frontend/website/src/Style.re [hunk 1]
```diff
@@ -2,6 +2,7 @@ module Colors = {
   let fadedBlue = `rgb((111, 167, 197));
   let white = Css.white;
   let hyperlink = `hsl((201, 71, 52));
+  let hyperlinkAlpha = a => `hsla((201, 71, 52, a));
```
Comment (bkase) [inline frontend/website/src/Style.re]: Wow this is awesome! This is why we love embedded DSLs! cc @bsansouci

Comment (bkase) [reply frontend/website/src/Style.re]: @Schmavery 	can you fix the azure_01 to be a function while you're here

File: frontend/website/src/TeamSection.re [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+        ~y=`zero,
+        ~blur=`px(4),
+        ~spread=`zero,
+        Style.Colors.greenShadow,
+      ),
+    ])
+  );
+
+module Member = {
+  let bgColor = `rgb((235, 235, 235));
```
Comment (bkase) [inline frontend/website/src/TeamSection.re]: can you add this to colors?

File: frontend/website/src/Style.re (hunk 1 above)
Comment (bsansouci) [reply frontend/website/src/Style.re]: \O/ ❤️

File: frontend/website/src/TeamSection.re (hunk 2 above)
Comment (Schmavery) [reply frontend/website/src/TeamSection.re]: it doesn't have a naaame

**Pull request merged**